web/src/components/tasks/business-formation/BusinessFormation.test.tsx

```diff
@@ -521,7 +521,7 @@ describe("<BusinessFormation />", () => {
     page.fillText("Address line1", "1234 main street");
     page.fillText("Address line2", "Suite 304");
     page.fillText("Address zip code", "08001");
-    page.selectByText("Address municipality", "Newark");
+    page.fillText("Address municipality", "Newark");
```

Comment:
same question here... I thought we are only updating the logic for registered agent.. does this ticket now include the other city field?